main.go

```diff
@@ -403,3 +422,24 @@ func getProjectFromToken(j string) string {
 	}
 	return t.ProjectID
 }
+
+func applyArgs(dryrun bool, file string) []string {
+	args := []string{
+		"apply",
+		"--record",
+	}
+
+	if dryrun {
+		args = append(args, "--dry-run")
+	}
+
+	args = append(args, "--filename")
+	args = append(args, file)
+
+	return args
+}
+
+func log(format string, a ...interface{}) (int, error) {
```

Comment:
Since callers don't care about the return type, you could make this function "void".